src/html.sortable.src.js

```diff
@@ -14,30 +14,170 @@
  */
 var dragging;
 var draggingHeight;
-var placeholders = $();
+var placeholders = [];
 var sortables = [];
+/**
+ * Get or set data on element
+ * @param {Element} element
+ * @param {string} key
+ * @param {*} value
+ * @return {*}
+ */
+var _data = function(element, key, value) {
+  if (value === undefined) {
+    return element && element.h5s && element.h5s.data && element.h5s.data[key];
+  } else {
+    element.h5s = element.h5s || {};
+    element.h5s.data = element.h5s.data || {};
+    element.h5s.data[key] = value;
+  }
+};
+/**
+ * Remove data from element
+ * @param {Element} element
+ */
+var _removeData = function(element) {
+  if (element.h5s) {
+    delete element.h5s.data;
+  }
+};
+var matches;
```

Comment:
Please add a docblock or comment here, so it is immediately clear why this switch statement is needed. (Only for the `_filter` below?)